pages/linux/fallocate.md

```diff
@@ -0,0 +1,16 @@
+# fallocate 
+
+> Reserve or deallocate disk space to files.
+> The utility allocates space without zeroing.
+
+- Reserve a file taking up 700MB of disk space:
+
+`fallocate --length {{700M}} {{path/to/file}}`
+
+- Shrink an already allocated file by 200MB:
+
+`fallocate --collapse-range --length {{200M}} {{path/to/file}}`
+
+- Deallocate 20MB of space after 100MB in a file:
```

Comment:
Should it be "Shrink" instead of "Deallocate" ? The command looks the same to me. It's just that here it is happening at an offset whereas the previous one shrinks from the end of the file. The description should be consistent in both cases.